_docs/_latest/configure/notifier/index.md

```diff
@@ -159,7 +192,7 @@ The following options are available.
 ```yaml
 notifier:
   violation:
-    cloud_scc:
+    cscc:
```

Comment:
Not here, but above, we should remove the reference to the Alpha as we'll only forget to come back and remove it once it's out of Alpha.